components/CollectionsPage/index.js

```diff
@@ -26,21 +26,21 @@ const CollectionsPage = () => {
 
             return (
               collection.publish ?
-                <Link href="/collections/[colId]" as={`/collections/${key}`}>
-                  <div className={scss.collections__tile} key={`collections-tile-${index + 1}`}>
-                    <img src={collection.image} />
+                <Link href="/collections/[colId]" as={`/collections/${key}`} key={`collections-tile-${index + 1}`}>
+                  <div className={scss.collections__tile}>
+                    <img src={collection.image} alt={`Collection ${collection.name}`}/>
                     <h2>{collection.name}</h2>
                     <p>{collection.description}</p>
                     <button className={scss.collections_button__active}>
                       <a>VIEW COLLECTION</a>
-                      <img src="static/icon/collections-page/button-arrow.svg" />
+                      <img src="static/icon/collections-page/button-arrow.svg" alt="View Collection"/>
```

Comment:
Because there's already text here that says VIEW COLLECTION, a screenreader would end up reading the same text twice. It's better to have an empty alt tag here.